specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/preview/2016-05-01/Alert.json

```diff
@@ -152,6 +152,52 @@
                     "application/json"
                 ]
             }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.InfrastructureInsights.Admin/regionHealths/{location}/alerts/{alertName}/remediate": {
+            "post": {
+                "x-ms-examples": {
+                    "Repairs an alert.": {
+                        "$ref": "./examples/Alert/Repair.json"
+                    }
+                },
+                "tags": [
+                    "Alerts"
+                ],
+                "x-ms-long-running-operation": true,
+                "description": "Repairs an alert.",
+                "operationId": "Alerts_Repair",
```

Comment:
This is not a good name.  Please change to Remediate.

Reply:
This was the guidance given to us by our PM (Charlie). Remediate is not an approved powershell verb and the UI and powershell should use repair.

Reply:
The route was updated to repair.